nncf/experimental/onnx/model_normalizer.py

```diff
@@ -101,13 +102,33 @@ def add_const_value_infos_to_graph(graph: onnx.GraphProto):
 
             nncf_logger.info(
                 'Successfully converted the model to the opset = {}'.format(modified_model.opset_import[0].version))
-        except (RuntimeError, ConvertError) as e:
+        except (RuntimeError, ConvertError):
             modified_model = model
-            nncf_logger.error("Couldn't convert target model to opset13. "
-                              "Models with opset < 13 is not supported by PTQ yet.")
-            raise ConvertError from e
+            nncf_logger.error(
+                "Couldn't convert target model to opset13. Use original model")
 
-        for i, node in enumerate(modified_model.graph.node):
+        return modified_model
+
+    @staticmethod
+    def replace_empty_node_name(model: onnx.ModelProto):
+        """
+        NNCFGraph.get_node_by_name() does not allow empty node names.
+        NNCF expects every node to have a unique name.
+        """
+        for i, node in enumerate(model.graph.node):
             if node.name == '':
                 node.name = node.op_type + '_nncf_' + str(i)
-        return modified_model
+
+        name_counter = defaultdict(int)
+
+        for i, node in enumerate(model.graph.node):
+            name_counter[node.name] += 1
+
+        for name, cnt in name_counter.items():
+            if cnt > 1:
+                raise RuntimeError(
+                    f"Node {name} occurs more than once ({cnt} times). "
+                    "NNCF expects every node to have a unique name."
+                )
```

Comment:
How about using counter like
```
if max(Counter(name for i, name in enumerate(model.graph.node)).values()) > 1:
    raise RuntimeError(
    ...
```

Reply:
Is there a way to avoid losing the duplicated node name if we use `Counter`?
I didn't see in your example.

Reply:
Ah I didn't see the name, cnt in RuntimeError. We should save the counter or dict as original.
Or how about remove the for loop and list all the duplicated names like
```
        name_counter = Counter(name for i, name in enumerate(model.graph.node))
        if max(name_counter.values()) > 1:
            raise RuntimeError(
                f"Nodes {[(name, cnt) for name, cnt in name_counter.items() if cnt > 1]} (name, counts) occurred more than once ..."
```

Reply:
Fixed.